docs/source/notebooks/Delineation_workflow.ipynb

```diff
@@ -25,7 +25,8 @@
     "import os\n",
     "\n",
     "# Set environment variable WPS_URL to \"http://localhost:9099\" to run on the default local server\n",
-    "url = os.environ.get(\"WPS_URL\", \"https://pavics.ouranos.ca/twitcher/ows/proxy/raven/wps\")\n",
+    "# url = os.environ.get(\"WPS_URL\", \"https://pavics.ouranos.ca/twitcher/ows/proxy/raven/wps\")\n",
+    "url = os.environ.get(\"WPS_URL\", \"http://localhost:9099/twitcher/ows/proxy/raven/wps\")\n",
```

Comment:
Can you remove your testing `localhost` line and restore the real raven from pavics.ouranos.ca?  And by the way, to avoid forgetting again next time, set the env `WPS_URL` to your `localhost` and you will not need to make this temporary modification that needs to be reverted again and again.